st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -88,7 +91,19 @@ def post(self, payload):
                                              notify=notify,
                                              context=context)
 
-        return str(execution.id)
+        result = {
```

Comment:
I think it's fine. We discussed this with @enykeev, and I can't see a realistic scenario where hubot-stackstorm will be older than st2. Even if that's the case, i.e. someone did an incomplete upgrade, we can just ask to do an `npm update`.

That's exactly why I'm asking to review this PR though: maybe you guys will see a case we overlooked where it can become a problem.